evalml/pipelines/components/transformers/preprocessing/text_transformer.py

```diff
@@ -22,6 +22,4 @@ def __init__(self, component_obj=None, random_seed=0, **kwargs):
 
     def _get_text_columns(self, X):
         """Returns the ordered list of columns names in the input which have been designated as text columns."""
-        text_column_vals = X.select('natural_language')
-        text_columns = list(text_column_vals.to_dataframe().columns)
-        return text_columns
+        return list(X.ww.select('NaturalLanguage').columns)
```

Comment:
@dsherry The string input here is the camel-case or snake-case representation of the Woodwork logical type class name. The only way this would change in Woodwork is if we rename the logical type class. You could use the class name directly instead of a string, but they shouldn't behave any differently...

Reply:
Understood, thanks.